core/src/mindustry/world/blocks/logic/CanvasBlock.java

```diff
@@ -388,14 +429,13 @@ public void draw(){
                     }).size(44f);
                 });
 
-                dialog.closeOnBack();
+                dialog.closeOnBack(dialog::hide);
```

Comment:
Why did you change this? Closing hides the dialog, you're just making it hide twice.

Reply:
oh right yeah I just did a substitution, thanks
fixed in https://github.com/Anuken/Mindustry/pull/10899/commits/4a02b42c291f67c295eb4b9211c4711a8134ba08